internal/sbi/api_policyauthorization.go

```diff
@@ -23,38 +23,55 @@ import (
 func (s *Server) getPolicyAuthorizationRoutes() []Route {
 	return []Route{
 		{
+			Name:    "PostAppSessions",
 			Method:  http.MethodPost,
 			Pattern: "/app-sessions",
 			APIFunc: s.HTTPPostAppSessions,
 		},
 		{
+			Name:    "DeleteEventsSubsc",
 			Method:  http.MethodDelete,
 			Pattern: "/app-sessions/:appSessionId/events-subscription",
 			APIFunc: s.HTTPDeleteEventsSubsc,
 		},
 		{
+			Name:    "UpdateEventsSubsc",
 			Method:  http.MethodPut,
 			Pattern: "/app-sessions/:appSessionId/events-subscription",
 			APIFunc: s.HTTPUpdateEventsSubsc,
 		},
 		{
+			Name:    "DeleteAppSession",
 			Method:  http.MethodPost,
 			Pattern: "/app-sessions/:appSessionId/delete",
 			APIFunc: s.HTTPDeleteAppSession,
 		},
 		{
+			Name:    "GetAppSession",
 			Method:  http.MethodGet,
 			Pattern: "/app-sessions/:appSessionId",
 			APIFunc: s.HTTPGetAppSession,
 		},
 		{
+			Name:    "ModAppSession",
 			Method:  http.MethodPatch,
 			Pattern: "/app-sessions/:appSessionId",
 			APIFunc: s.HTTPModAppSession,
 		},
+		{
+			Name:    "PcscfRestoration",
+			Method:  http.MethodPost,
+			Pattern: "/app-sessions/pcscf-restoration",
+			APIFunc: s.HTTPPcscfRestoration,
+		},
 	}
 }
 
+// PcscfRestoration - Indicates P-CSCF restoration and does not create an Individual Application Session Context
+func (s *Server) HTTPPcscfRestoration(c *gin.Context) {
+	c.JSON(http.StatusNotImplemented, nil)
```

Comment:
Maybe changing the second parameter of c.JSON to `gin.H{}` would be better.
`gin.H{}` returns an empty JSON object ({}). As same as the others' changing way (refer to AMF).